src/components/generic/OptionsList/Option.scss

```diff
@@ -18,7 +18,9 @@ $C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-option;
   }
 
   .#{$C}__check {
+    background: var(--wxc-options-list-item--selected--background);
```

Comment:
add a minus: --wxc-options-list--item--selected--background
same for the other 2 below